src/components/Header/index.tsx

```diff
@@ -1,11 +1,9 @@
 import { Trans } from '@lingui/macro'
 import useScrollPosition from '@react-hook/window-scroll'
 import { useWeb3React } from '@web3-react/core'
```

Comment:
wallet was in old header, I removed it.  I am worried we will forget it's in here and the logic to switch between the two isn't simple.  So, it's now only accessible through new nav